spec/helpers/vacancies_helper_spec.rb

```diff
@@ -69,27 +69,35 @@
       allow(vacancy).to receive(:state).and_return('copy')
       allow(vacancy).to receive(:job_title).and_return('Test job title')
 
-      expect(page_title(vacancy, school)).to eql(I18n.t('jobs.copy_job_title', job_title: 'Test job title'))
+      expect(page_title(vacancy)).to eql(I18n.t('jobs.copy_job_title', job_title: 'Test job title'))
     end
 
     it 'returns create a job title if vacancy state is create' do
       allow(vacancy).to receive(:published?).and_return(false)
       allow(vacancy).to receive(:state).and_return('create')
+      allow(vacancy).to receive(:school_or_school_group).and_return(school)
+      allow(school).to receive(:name).and_return('school')
```

Comment:
Can you use a more realistic school name, to better show your intention with this test?

Reply:
Done!